sqlite/sqlite-bundled/build.gradle

```diff
@@ -55,19 +58,23 @@ def SQLITE_COMPILE_FLAGS = [
         "-DSQLITE_THREADSAFE=2",
 ]
 
-// Copy SQLite sources into a build directory. Even though this is not needed, it will be
-// nice to have them there so that we can swap them with SQLite amalgamation for Github.
-// TODO(b/306669673): Support GitHub builds
-def copySqliteSourcesTask = tasks.register("copySqliteSources", Copy) {
-    from(new File(AndroidXConfig.getExternalProjectPath(project), "sqlite/dist/orig")) {
-        include "sqlite3.c"
-        include "sqlite3.h"
+// see PrepareSqliteSourcesTask.groovy for implementation details of the register method
+def prepareSqliteSourcesTask = registerPrepareSqliteSourcesTask(
+        project, // project
+        "prepareSqliteSourcesTask" // task name
```

Comment:
you can use named arguments in groovy
registerPrepareSqliteSourcesTask(project: project, name: "prepareSqliteSourcesTask")

Reply:
not sure about that.
as far as i can see, it requires me to use a Map as an argument in the function declaration, which would make the function declaration less useful.

https://docs.groovy-lang.org/latest/html/documentation/#_named_parameters_2

am i missing something ?

Reply:
You are right, I misunderstood how groovy does it. we can leave it as-is